src/mame/namco/namcos21_3d.cpp

```diff
@@ -182,68 +170,40 @@ void namcos21_3d_device::rendertri(const n21_vertex *v0, const n21_vertex *v1, c
 
 		e2.x = v0->x;
 		e2.z = v0->z;
-		crop = -ystart;
-		if (crop > 0)
+
+		if (ystart < 0)
 		{
-			e2.x += dx2dy * crop;
-			e2.z += dz2dy * crop;
+			e2.x += dx2dy * -ystart;
+			e2.z += dz2dy * -ystart;
 		}
```

Comment:
Again, adding the blank line here isn’t helpful – it’s all part of setting up edge `e2`, not separate logical operations.

Reply:
Done.